README.md

```diff
@@ -328,9 +328,9 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Guidewire](https://careers.guidewire.com/job/76acec73-8be6-5e85-992c-6f4afca84434) | Bedford, MA; or Exton, PA; or Birmingham, AL; or San Mateo, CA | Summer 2023 Software Engineering Intern - Multiple Locations |
 | [84.51°](https://app.ripplematch.com/job/84.51°/2344fff3/) | Cincinnati, Chicago | [Software Engineer - Internship Program 2023](https://app.ripplematch.com/job/84.51°/2344fff3/)|
 | [Akamai](https://app.ripplematch.com/job/Akamai/c41ee243/) | Various - Cambridge, Portland, Santa Clara, Fort Lauderdale | [Summer 2023 Intern - Software Engineer](https://app.ripplematch.com/job/Akamai/c41ee243/)|
-| [Amplitude](https://app.ripplematch.com/job/Amplitude/3d6b2d5e/) | Vancouver, Canada | [Software Engineer Intern](https://app.ripplematch.com/job/Amplitude/3d6b2d5e/)|
+| [Amplitude](https://app.ripplematch.com/job/Amplitude/3d6b2d5e/) | Vancouver, Canada | **🔒 Closed 🔒** [Software Engineer Intern](https://app.ripplematch.com/job/Amplitude/3d6b2d5e/)|
```

Comment:
for some reason icant edit the file so leaving comment instead.

Please follow closed internship format (no link in left column)
<img width="953" alt="image" src="https://user-images.githubusercontent.com/36709118/191868907-e5b17964-53c5-49a0-b8b3-e7dabb7b23c3.png">

Reply:
done